5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
As additional reference see the official PHP image here. They don't use an external script for a way more complex install process: https://github.com/docker-library/php/blob/master/5.6/fpm/Dockerfile#L48.